client/termdb/TermdbVocab.js

```diff
@@ -868,6 +868,23 @@ export class TermdbVocab extends Vocab {
 		return await dofetch3('termdb', { headers, body })
 	}
 
+	async getLowessCurve(opts) {
```

Comment:
@airenzp may consider doing this on the client-side, for example using https://github.com/HarryStevens/d3-regression? Since the data points are already available from the server response, it seems that a client-side stat library may be more appropriate.